app-emulation/libvirt-glib/libvirt-glib-4.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson vala
+
+DESCRIPTION="GLib and GObject mappings for libvirt"
+HOMEPAGE="https://libvirt.org/ https://gitlab.com/libvirt/libvirt-glib/"
+SRC_URI="https://libvirt.org/sources/glib/${P}.tar.xz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="+introspection +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+BDEPEND="
+        dev-util/glib-utils
+        dev-util/gtk-doc-am
+        >=dev-util/intltool-0.35.0
+        virtual/pkgconfig
+        vala? ( $(vala_depend) )"
```

Comment:
I like to put the trailing `"` on its own line. It helps minimize the diff if a dependency is added in the future.